src/platforms/flutter/upload-debug.mdx

```diff
@@ -24,6 +24,12 @@ Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication
 
 ##  Uploading for Android NDK
 
+<Note>
+
+Native symbolication on Android requires a specific configuration described on the [troubleshooting](/platforms/flutter/troubleshooting/#native-symbolication-on-android) page.
+
+</Note>
+
```

Comment:
This location is better to surface the required set up for Android symbolication.

```suggestion
Native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` in your `gradle.properties` file.
```